### DISCUSSION TITLE
Attention pour la procédure Rebase

### DISCUSSION THREAD

**Discussion by CollRen** [582-41E-MA/projet-web-2-CollRen#114] [Rebases]
1. Accepter Pull Request en tout premier s'il y en a un
2. Pull dev
3. Rebase votre branche sur dev
4. Push votre branche

**Comment by NabilDjerroud** [comment]
ok @CollRen , y a t il eu un problème ?

**Reply by CollRen** [reply]
Un peu en début de semaine lors de la remise les filtres n'étaient pas fonctionnels...

@NabilDjerroud je serais disponible pour répondre à tes questions après 21h ou demain; texte-moi directement 514 842-0313
